src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/set_display_layout_request.cc

```diff
@@ -154,7 +152,15 @@ void SetDisplayLayoutRequest::on_event(const event_engine::Event& event) {
                               hmi_response::templates_available);
           }
         }
+
+        // In case of successful response warn user that this RPC is deprecated
+        result_code = hmi_apis::Common_Result::WARNINGS;
```

Comment:
This warning message should only be relayed to apps using RPC version that this rpc is deprecated for. ie apps >= 6.0 && apps < 7.0

Reply:
it was discussed with @kshala-ford that this message needs to be present regardless app version

Reply:
@mked-luxoft Sorry but I don't agree with this response. Sending a warning result code could break an apps implementation that negotiates an RPC spec version where SetDisplayLayout is completely valid. According to the versioning in the API, this RPC is only deprecated for apps using rpc spec version 6.x, therefore only 6.x apps should be receiving the warning.

Reply:
Hello @JackLivio agree. To be clear here: For version < 6.0 the result code should be `SUCCESS`. For version >= 6.0 it should be `WARNINGS`. 

There should not be a condition like `apps < 7.0` as this is not specified.

Reply:
@JackLivio Thank you. Fixed in f8085a5316b7b0deef654c69489a63b33d444a7f